api/devicerepository.proto

```diff
@@ -303,11 +307,51 @@ message ListEndDeviceModelsResponse {
   repeated EndDeviceModel models = 1;
 }
 
-message MessagePayloadFormatter {
+message EncodedMessagePayload {
+  uint32 f_port = 1 [(validate.rules).uint32.lte = 255];
+  bytes frm_payload = 2 [(gogoproto.customname) = "FRMPayload"];
+  repeated string warnings = 3;
+  repeated string errors = 4;
+}
+
+message DecodedMessagePayload {
+  google.protobuf.Struct data = 1;
+  repeated string warnings = 2;
+  repeated string errors = 3;
```

Comment:
Should we set validation rules for maximum length (both of the list, and of each list element)?